src/etl/ReportingETL.cpp

```diff
@@ -1159,5 +1343,23 @@ ReportingETL::ReportingETL(
         if (cache.contains("page_fetch_size") &&
             cache.at("page_fetch_size").is_int64())
             cachePageFetchSize_ = cache.at("page_fetch_size").as_int64();
+        if (cache.contains("clio_peers") && cache.at("clio_peers").is_array())
```

Comment:
I kind of think this should just be called "peers"

Reply:
agree, simpler yet still makes total sense